<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reserveTicketsMenu</class>
 <widget class="QWidget" name="reserveTicketsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*--------------------------------Backdround color--------------------------------*/
#LoginMenu{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(23, 30, 34, 255), stop:0.5 rgba(30, 40, 45, 255), stop:1 rgba(23, 30, 34, 255));
}
/*--------------------------------Button--------------------------------*/

QPushButton{
	
	background-color: rgba(242, 150, 47, 220);
	
	border:none;
	border-radius:10px;

	color:#1b2327;
	font-family:  &quot;Calibri Bold&quot;;
	font-size: 17px;

}

QPushButton:hover:!pressed{
	background-color: #455A64;
color:#EEEEEE;

}

QPushButton:hover:pressed{
	background-color: #37474F;
color:#DDDDDD;
}


/*--------------------------------Check box--------------------------------*/
QCheckBox {
	color:#999999;
	font: italic 9pt &quot;Open Sans Light&quot;;
	border-radius:6px;
}

QCheckBox::indicator:checked {
    background: #198C49;
	border-radius:6px;
}

QCheckBox::indicator:unchecked {
    background:#990329;
	border-radius:6px;
}

QCheckBox::indicator:checked:hover {
  	border: 2px solid #198C49;
    background: #15733C;
	border-radius:6px;
}

QCheckBox::indicator:unchecked:hover {
	border: 2px solid #990329;
	background:#800322;
	border-radius:6px;
}

/*--------------------------------Line edit--------------------------------*/

QLineEdit{
	background-color:#37474F ;

	border-style: solid;
	border-width: 2px;
	border-color: #455A64;
	border-radius: 10px;

	color:#ECEFF1;
	font-family: &quot;Calibri&quot;;
	font-size: 17px;

	padding-left:10px;
	padding-right:10px;
}


QLineEdit:hover{
	background-color: #263238;
}

QLineEdit:focus{
	background-color: #1c2428;
}

/*--------------------------------Line edit--------------------------------*/

QCheckBox{
	color:#78909C;
	font-family: &quot;Calibri&quot;;
	font-size: 15px;
	background-color: none;

}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Bought tickets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableView" name="boughtTicketsTable"/>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <property name="horizontalSpacing">
            <number>7</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="ownerLname">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Owner last name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>8</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="buyButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Buy</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="returnButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Return</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ownerFname">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Owner first name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Reserved tickets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableView" name="reservedTicketsTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tickets history</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTableView" name="unActiveTicketsTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
